### PR TITLE
Adds a flash pen disguised as a medical holopen

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Tators in the medical field can get a flash disguised as a medical holopen now! it costs 6 tc

## Why It's Good For The Game
more stealth opprotunities?

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: flash holopen tator item
/:cl:

### CODE REVIEW

**Comment by EvilDragonfiend** [comment]
Shouldn't it be specifically a holopen? Adding a specific item doesn't make you stealthy, and it should use chameleon feature instead.

**Comment by FriendlyContractor** [comment]
Its worse than easily available "common"l flash. Very expensive for what it is and gives away that you are a traitor.

File: code/modules/assembly/flash.dm [hunk 1]
```diff
@@ -315,6 +315,43 @@
...
+			to_chat(user, "<span class='warning'>[src] is not ready yet!</span>")
+			return
+		var/T = get_turf(target)
+		if(locate(/mob/living) in T)
+			new /obj/effect/temp_visual/medical_holosign(T,user) //produce a holographic glow
+			holo_cooldown = world.time + 100
+			return
+
+
+/obj/item/assembly/flash/pen/suicide_act(mob/living/carbon/human/user)
```
Comment (itsmeow) [inline code/modules/assembly/flash.dm]: Why add this? Is it from the parent holopen? Honestly just remove if so.

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -1992,6 +1992,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	restricted_roles = list(JOB_NAME_MEDICALDOCTOR, JOB_NAME_CHIEFMEDICALOFFICER, JOB_NAME_ROBOTICIST, JOB_NAME_PARAMEDIC, JOB_NAME_BRIGPHYSICIAN)
 	cost = 3
 
+/datum/uplink_item/role_restricted/holoflash
+	name = "Penlight Flash"
+	desc = "Modifying the standard issue penlight, we were able to make it emit light similar to that of a disorienting flash. Be careful, as it also requires flash bulbs, and it only comes with one."
+	item = /obj/item/assembly/flash/pen
+	restricted_roles = list(JOB_NAME_MEDICALDOCTOR, JOB_NAME_CHIEFMEDICALOFFICER, JOB_NAME_ROBOTICIST, JOB_NAME_PARAMEDIC, JOB_NAME_BRIGPHYSICIAN)
+	cost = 6
```
Comment (itsmeow) [inline code/modules/uplink/uplink_items.dm]: Cost is a bit high, maybe 4-5TC?

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/modules/uplink/uplink_items.dm (hunk 2 above)
Comment (ivanmixo) [reply code/modules/uplink/uplink_items.dm]: This is barely worth 2, lol

**Review by itsmeow — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
Why would I use this over the infinite hypnoflash?

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
This is just a flash in the uplink really, I don't think its unique enough unlike stuff like the rad scanner which has its own special effects. Already not a fan of flashes due to their function being instant combat victory or being entirely useless. This is too expensive none the less

**Comment by Penwin0** [comment]
Quite expensive for what is a retextured flash, a bit of a boring item for a job-special or traitor item in general. This PR has been sitting here since August without the issues pointed out being resolved

**Pull request closed without merging**